front/components/vaults/VaultDataSourceViewContentList.tsx

```diff
@@ -279,21 +289,6 @@ export const VaultDataSourceViewContentList = ({
           await mutateDataSourceViewContentNodes();
         }}
       />
-      {connector && (
-        <ConnectorPermissionsModal
-          owner={owner}
-          connector={connector}
-          dataSource={dataSourceView.dataSource}
-          isOpen={showConnectorPermissionsModal}
-          onClose={() => {
-            setShowConnectorPermissionsModal(false);
-          }}
-          plan={plan}
-          readOnly={false}
-          isAdmin={isAdmin}
-          dustClientFacingUrl={dustClientFacingUrl}
-        />
-      )}
```

Comment:
@JulesBelveze I removed this part, the modal wsa duplicated - did not see where this one was used ?